Examples/rc-maestro/maestro/utils/confirm_purchase.yaml

```diff
@@ -34,5 +34,7 @@ env:
       - tapOn:
           text: "Subscribe"
       - takeScreenshot: "${SCREENSHOT_PREFIX} - IAP Confirmation"
+      - waitForAnimationToEnd
       - tapOn:
           text: "OK"
+          waitUntilVisible: true
```

Comment:
Hmm I would assume this would be true by default? In that case, it wouldn't be needed I think.